### ISSUE TITLE
MadGraph tarballs on cvmfs

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/genproductions#3272]
Rather than downloading the MG5 source tarball from the web every time:
https://github.com/cms-sw/genproductions/blob/d0d03f89778a7895a8164dfa8f513c32d3419df2/bin/MadGraph5_aMCatNLO/gridpack_generation.sh#L183
https://github.com/cms-sw/genproductions/blob/d0d03f89778a7895a8164dfa8f513c32d3419df2/bin/MadGraph5_aMCatNLO/gridpack_generation.sh#L141,
which leaves the possibility that the job will fail because of a transient connectivity issue,
why not host the tarballs for the MG versions that we use on cvmfs?

**Comment by agrohsje** [comment]
Hi Kevin, in central production we had the same issue with cvmfs. Jobs are failing because gridpacks are not accessible on cvmfs. Did you encounter problems with the web?  Me neither, despite producing hundreds of gridpacks.

**Comment by kpedro88** [comment]
> Did you encounter problems with the web?

Yes, I had gridpack creation jobs fail yesterday because the wget call (noted above) did not succeed.

**Comment by sihyunjeon** [comment]
@agrohsje i agree with this change though, since the web is at the stage where it is impossible to clean up at this point. why don't we clean things up by minimally changing the logics?

**Comment by agrohsje** [comment]
I think there is nothing wrong with moving. It is just a question of priorities and other requests. If someone has time to fix, no problem. It is also not a lot of work, but too often not a lot of work is a lot of work.
